### DISCUSSION TITLE
how to change text colour?

### DISCUSSION THREAD

**Discussion by MikuHoney** [Monika-After-Story/MonikaModDev#9821] [QA - code and content creation (technical)] [answered]
hi, im making my ui green, ive got everything green.. but the text like monikas name and the "talk, music extra play" stuff, how do i change that? and what program would i need, if any? thankyou
oh, and where is the image file that contains the pink on attached image? i need to change that too haha
![image](https://user-images.githubusercontent.com/121546213/209788808-5c3ef704-f32d-4a32-9d1a-336e7a0b15e0.png)

**Comment by Booplicate** [comment, marked as answer]
You will need to research how the engine works. For example, [this](https://www.renpy.org/dev-doc/html/style_properties.html). Don't replace files, make overrides.

**Reply by MikuHoney** [reply]
whenever i try view the styles.rpyc file it shows up as if its corrupted? for example :RENPY RPC2☺   .   !!  ☻   O!  !!              xÚÝ]•`∟å•¶♦ØfmÀ€cZ€a
so on and so forth.. just lots of that. i used note pad and a file viewing website, and both are corrupted it seems. am i doing something wrong? is this the right file?

**Reply by Wingdinggaster656** [reply]
`.rpyc` isn't `.rpy`. `rpyc` is compiled.
Although rpyc files may be decompiled, you do not need to do so for MAS - MAS is open source. You should get the files you need from this Github repository.

**Reply by MikuHoney** [reply]
ohh okay thank you so much!

**Reply by MikuHoney** [reply]
![image](https://user-images.githubusercontent.com/121546213/209899740-93125d83-7395-4fe0-bcf5-972d91f3428a.png)
thank you so much everyone!! :) doesnt it look awesome!!

**Reply by Booplicate** [reply]
> whenever i try view the styles.rpyc file

You shouldn't edit it btw, you create your own submod scripts and make overrides there.
